src/api/kube.ts

```diff
@@ -1743,6 +1745,22 @@ export class KubeHelper {
     }
   }
 
+  async patchClusterServiceVersion(namespace: string, name: string, jsonPatch: any[]): Promise<ClusterServiceVersion> {
+    const customObjectsApi = KubeHelper.KUBE_CONFIG.makeApiClient(CustomObjectsApi)
+
+    const requestOptions = {
+      headers: {
+        'content-type': 'application/json-patch+json'
+      }
+    }
+    try {
+      const response = await customObjectsApi.patchNamespacedCustomObject('operators.coreos.com', 'v1alpha1', namespace, 'clusterserviceversions', name, jsonPatch, undefined, undefined, undefined, requestOptions)
```

Comment:
Not clear why `jsonPath` is array if `patchNamespacedCustomObject` accept object.

Reply:
Because by the [spec](http://jsonpatch.com/) json patch is an array of patch objects and such type of patching is supported by Kubernetes.
The reason why I used this type of patching is that it doesn't require updating of other fields, unlike `application/merge-patch+json`

Reply:
I see.